tests/yam/validate/validate_relative_urls.pm

```diff
@@ -0,0 +1,29 @@
+# SUSE's openQA tests
```

Comment:
https://openqa.suse.de/tests/18230601/modules/validate_deployed_files/steps/1/src
this module should be the one that we use everywhere if I get it correctly, we don't need new module for relative url, you could edit that one and then provide test data for that case with 2 files too. Other than that looks good.